tensorwaves/physics/helicityformalism/amplitude.py

```diff
@@ -136,31 +139,27 @@ def create_element(self, recipe: dict) -> Callable:
             self, recipe, kinematics=self._kinematics
         )
 
-    def register_element_builder(
-        self,
-        element_name: str,
-        builder: Callable[["IntensityBuilder", dict], Callable],
-    ) -> None:
-        """Register builder functions for a specific operation.
-
-        Allows user to inject personalized building code which will be call once
-        an element with corresponding name is found.
-
-        """
-        if element_name in self._registered_element_builders:
-            logging.warning(
-                "Overwriting previously defined builder for %s", element_name
-            )
-        self._registered_element_builders[element_name] = builder
-
     def create_dynamics(
         self,
         decaying_state_name: str,
         dynamics_properties: "DynamicsProperties",
     ) -> Callable:
         """Create a dynamics function callable."""
         dynamics_builder = self._get_dynamics_builder(decaying_state_name)
-        return dynamics_builder(dynamics_properties)
+
+        decay_dynamics = self._dynamics[decaying_state_name]
+        kwargs = {}
+        if "FormFactor" in decay_dynamics:
+            kwargs.update({"FormFactor": decay_dynamics["FormFactor"]["Type"]})
```

Comment:
Does the key `"FormFactor"` agree with PEP conventions? If I understand `kwargs` correctly, this means one of the arguments of the callable will be `FormFactor`, not `formfactor`.

Reply:
good point

Reply:
I improved the code further. Check it out and resolve if you are satisfied